kernel/solo5.h

```diff
@@ -102,15 +129,17 @@ int solo5_blk_rw(void);
  */
 
 /*
- * Writes (n) bytes from (*buf) to the console. Returns number of bytes
- * actually written.
+ * Writes (size) characters from (*buf) to the console.
+ *
+ * Always succeeds, however the implementation may truncate messages (e.g. due
```

Comment:
this is not entirely true: if `write` returns `< 0`, an assertion is raised (in the ukvm implementation): https://github.com/mato/solo5/blob/13e7a06cfcb24452d0e8113057cf4719f7ca2915/ukvm/ukvm_core.c#L103